### PR TITLE
[AVOID-169] rs_launch: allow to properly set the config_path file as a parameter

### PR DESCRIPTION
**Pull request by TSC21**
Jira task: https://auterion.atlassian.net/browse/AVOID-169.

And so to load the config file stored in the shared dir.

To test:

```sh
ros2 launch realsense2_camera rs_launch.py config_file:="'d435i.yaml'"
```

### CODE REVIEW

File: realsense2_camera/config/d455_skynode.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+camera1:
+    camera:
+      ros__parameters:
+        device_type: ''
```
Comment (TSC21) [inline realsense2_camera/config/d455_skynode.yaml]: maybe we can hardcode the 'd455'?

File: realsense2_camera/config/d455_skynode.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        serial_no: ''
+        usb_port_id: ''
+
+        config_file: "config/camera-depth-filter-settings.json"
+
+        depth_fps: 10.0
```
Comment (TSC21) [inline realsense2_camera/config/d455_skynode.yaml:10]: should we keep it at 15 and let the algorithm decide at what rate to process it?

File: realsense2_camera/config/d455_skynode.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        infra_fps: 10.0
+        infra_width: 848
+        infra_height: 480
+        infra_qos: "SENSOR_DATA"
+
+        enable_accel: false
+        enable_color: false
+        enable_depth: true
+        enable_gyro: false
+        enable_infra1: true
```
Comment (TSC21) [inline realsense2_camera/config/d455_skynode.yaml:24]: do we need infra1? what about infra2? you know the difference?

File: realsense2_camera/launch/d455_skynode.py [hunk 4]
```diff
@@ -0,0 +1,157 @@
+# Copyright (c) 2018 Intel Corporation
```
Comment (TSC21) [inline realsense2_camera/launch/d455_skynode.py]: Do we need this now? :P

File: realsense2_camera/config/d455_skynode.yaml (hunk 2 above)
Comment (bastianhjaeger) [reply realsense2_camera/config/d455_skynode.yaml:10]: No. Not possible on  the skynode.

File: realsense2_camera/config/d455_skynode.yaml (hunk 1 above)
Comment (bastianhjaeger) [reply realsense2_camera/config/d455_skynode.yaml]: Like this?

File: realsense2_camera/config/d455_skynode.yaml (hunk 3 above)
Comment (bastianhjaeger) [reply realsense2_camera/config/d455_skynode.yaml:24]: It is the left and the right camera. We do not need any of them, but it comes in handy for testing to have a gray 8bit video stream

File: realsense2_camera/config/d455_skynode.yaml (hunk 2 above)
Comment (bastianhjaeger) [reply realsense2_camera/config/d455_skynode.yaml:10]: On USB2 we only have
```
    Depth	  848x480	@ 10Hz	   Z16
    Depth	  848x480	@ 5Hz	   Z16
```

**Review by TSC21 — COMMENTED** [review]
Perfect thanks @bastianhjaeger!

**Pull request merged**